lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js

```diff
@@ -18,9 +18,10 @@ module.exports = [
         'first-contentful-paint': {
           numericValue: '>2000',
         },
-        'first-cpu-idle': {
-          numericValue: '>2000',
-        },
+        // TODO: ?
```

Comment:
Should I replace these smoke expectations with something else?

Reply:
nah. FCI is basically TTI and TTI is already there.

Reply:
delete?